Sprint 2/MemorizeWordsGame.java

```diff
@@ -0,0 +1,39 @@
+package randomWordGame;
+
+import java.util.ArrayList;
+
+/**
+ * Class that begins game.
+ * @author Christine
+ *
+ */
+public class MemorizeWordsGame {
+
+	/**
+	 * Begins game with all of the user's game selections.
+	 * @throws InterruptedException thrown when sleep is interrupted before/during execution.
+	 */
+	
+	//private static ArrayList<String> theList = new ArrayList<String>();
+	public static void startGame() throws InterruptedException {
+		
+		System.out.println("Game starting in: \n3..." );
+		Thread.sleep(1000);
+		System.out.println("2...");
+		Thread.sleep(1000);
+		System.out.println("1...\n");
+		Thread.sleep(1000);
+		
+		//timer start
+		ArrayListOps.displayWords(MenuSystem.gameArrayList);
+		
+		//timer end 

```

Comment:
It seems like you forgot to add some functions, unless these functions are supposed to be for Sprint 3.